source/themes/simple/SimpleRadio.scss

```diff
@@ -0,0 +1,77 @@
+@import "theme";
+
+// OVERRIDABLE CONFIGURATION VARIABLES
+
+$radio-size: 23px !default;
+$radio-color: white !default;
+$radio-border-color: #294B70 !default;
+
+$radio-border-disabled-color: $theme-color-disabled !default;
+$radio-label-color: #5e6066 !default;
+$radio-label-disabled-color: $theme-color-light-grey !default;
+
+.root {
+  display: flex;
+  position: relative;
+  &:hover {
+    cursor: pointer;
+  }
+  &.disabled {
+    &:hover, .label:hover {
+      cursor: default;
+    }
+    .label {
+      color: $radio-label-disabled-color;
+    }
+  }
+}
+
+.input {
+  height: 0;
+  opacity: 0;
+  overflow: hidden;
+  position: absolute;
+  width: 0;
+}
+
+.bubble {
```

Comment:
Shouldn't that be called `.radio` or `.circle`?

Reply:
haha I guess I had `bubble` in my mind for some reason ;)